### DISCUSSION TITLE
How to generate jwt key

### DISCUSSION THREAD

**Discussion by MuhammadNaeem42** [reconmap/reconmap#146] [Q&A] [answered]
![Screenshot from 2024-09-28 08-47-40](https://github.com/user-attachments/assets/05c6deb3-9a8c-40e3-90d9-7a00ebf1d2c0)

**Comment by santiagolizardo** [comment, marked as answer]
The new version of the agent (reconmapd) does not require a key but client credentials instead (id, password). See [config-agent.json](https://github.com/reconmap/reconmap/blob/master/config-agent.json) as an example. Thanks
